packages/0x.js/src/order_watcher/expiration_watcher.ts

```diff
@@ -23,7 +23,15 @@ export class ExpirationWatcher {
         this._orderExpirationCheckingIntervalMs =
             expirationMarginIfExistsMs || DEFAULT_ORDER_EXPIRATION_CHECKING_INTERVAL_MS;
         const scoreFunction = (orderHash: string) => this._expiration[orderHash].toNumber();
-        const comparator = (lhs: string, rhs: string) => scoreFunction(lhs) - scoreFunction(rhs);
+        const comparator = (lhsOrderHash: string, rhsOrderHash: string) => {
+            const lhsExpiration = this._expiration[lhsOrderHash].toNumber();
```

Comment:
are we still using `scoreFunction` anywhere? should we be calling that here?

Reply:
No. Good catch